lib/ansible/modules/net_tools/netbox/netbox_interface.py

```diff
@@ -108,6 +108,13 @@
     choices: [ absent, present ]
     default: present
```

Comment:
Please add `virtual_machine` as a sub option in `data` since it doesn't use the `device` suboption for this and that should take care of documenting the change.

It might be able to be just kept `device` as well and the only thing they need to change is the `app`, but I'll leave that up to you.

Otherwise, tested and it works.

Reply:
Thank you for the prompt review. 

I added the documentation for virtual_machine as sub-option of data, and added clarification to data.device as well.